Library/Formula/avidemux.rb

```diff
@@ -124,6 +115,7 @@ def install
           -DPLUGIN_UI=#{plugin}
           -DAVIDEMUX_LIB_DIR=#{lib}
           -DAVIDEMUX_SOURCE_DIR=#{buildpath}
+          -DCMAKE_OSX_SYSROOT=#{MacOS.sdk_path}"
```

Comment:
Currently getting following message:

```
==> cmake -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/avidemux/2.6.12 -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev -DPLUGIN_UI=COMMON -DAVIDEMUX_LIB_DIR=/usr/local/Cellar/avidemux/2.6.12/lib -DAVIDEMUX_SOURCE_DIR=/private/tmp/avidemux20160312-35549-ljigys/avidemux_2.6.12 -DCMAKE_OSX_SYSROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk" ../avidemux_plugins
CMake Warning at /usr/local/Cellar/cmake/3.5.0/share/cmake/Modules/Platform/Darwin-Initialize.cmake:132 (message):
  Ignoring CMAKE_OSX_SYSROOT value:

   /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk"

  because the directory does not exist.
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.5.0/share/cmake/Modules/CMakeSystemSpecificInitialize.cmake:18 (include)
  CMakeLists.txt
```